### DISCUSSION TITLE
Can someone show me how to change the color of the cursor ?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#4107] [Q&A]
I tried the following code, but it didn't work.

```lua
vim.api.nvim_set_hl(0, "Cursor", { fg = "#FF0000" })
vim.api.nvim_set_hl(0, "iCursor", { fg = "#FF0000" })
```

**Comment by dotfrag** [comment]
There is an example on tokyonight on how to override highlights, but I don't think Cursor is the highlight you want.

https://github.com/folke/tokyonight.nvim?tab=readme-ov-file#-overriding-colors--highlight-groups

I'm not sure which one it is though.
